mask-parser/src/parser.rs

```diff
@@ -56,9 +56,12 @@ pub fn parse(maskfile_contents: String) -> Maskfile {
             }
             End(tag) => match tag {
                 Tag::Header(_) => {
-                    let (name, required_args) = parse_command_name_and_required_args(text.clone());
+                    // let (name, required_args) = parse_command_name_and_required_args(text.clone());
```

Comment:
There's a few leftover/dead comments in this PR, can you remove them all?

Reply:
Sorry about that. I went back through all files I made changes to and I believe I fixed everything.